src/dmd/mtype.h

```diff
@@ -549,10 +549,10 @@ enum RET
 
 enum TRUST
 {
-    TRUSTdefault = 0,
-    TRUSTsystem = 1,    // @system (same as TRUSTdefault)
-    TRUSTtrusted = 2,   // @trusted
-    TRUSTsafe = 3,      // @safe
+    default_ = 0,
+    system = 1,    // @system (same as TRUST.default)
+    trusted = 2,   // @trusted
+    safe = 3,      // @safe
```

Comment:
Fails with:
```
dmd/mtype.h:553: error: 'system' redeclared as different kind of symbol
/usr/include/stdlib.h:113: error: previous declaration of 'int system(const char*)'
```

Maybe suffix all enum members with an underscore?

Reply:
The problem was that enum doesn't create its own scope in C++ 98.